provider/calc.go

```diff
@@ -139,6 +148,16 @@ func (o *calcProvider) floatGetter() (float64, error) {
 			}
 		}
 
+	case o.abs != nil:
+		v, err := o.abs()
+		if err != nil {
+			return 0, fmt.Errorf("abs: %w", err)
+		}
+		res = v
```

Comment:
res = math.Abs(v)

Reply:
Okay, meine Go-Kenntnisse sind leider sehr beschränkt.
Hatte mich am "sign" orientiert, wo es meiner Meinung nach auch sehr umständich umgesetzt ist.

Reply:
Hast du einen Vorschlag?

Reply:
Nein.
Andere Sprachen haben üblicherweise ein math.Sign(), welches man aber [in Go](https://pkg.go.dev/math) offensichtlich "vergessen" hat.
Es ginge sicherlich mit math.Copysign(), was vielleicht etwas leichter zu lesen ist als das "map[bool]float64{...}[]" - aber auch da müsste der Fall 0 gesondert behandelt werden. Gleiches gilt für den eventuell auch naheliegende Umsetzung:
`res = v / math.Abs(v)`

Von daher: never change a working code.

Reply:
No, this is not exactly the same...

With sign() you get the following results:
-123 => -1
0 => 0
45 => 1

With math.Copysign(1, 0) you still get 1, not 0.